src/components/Navigation/NavBar.scss

```diff
@@ -1,27 +1,29 @@
 @import '../../assets/css/fonts';
 @import '../../assets/css/colors';
+@import '../../assets/css/nav';
 
-.fontHeader {
+.fontNavBar {
   font-family: $font-default;
   font-weight: 300;
   color: $color-green-dark;
 }
 
 .navBar {
```

Comment:
Går det an å overskrive default navbar i stedet for å ha klassene .fontNavBar og .navBar her?